plugins/services/src/js/containers/tasks/TaskTable.js

```diff
@@ -327,19 +327,21 @@ class TaskTable extends React.Component {
     const unhealthy = task.health === TaskHealthStates.UNHEALTHY;
     const unknown = task.health === TaskHealthStates.UNKNOWN;
 
+    const failing = ["TASK_ERROR", "TASK_FAILED", "TASK_KILLING"].includes(
```

Comment:
Moreover on the line 351 we're using `failing` variable to assign `danger` css class, I'm not sure if it's correct